migrations.go

```diff
@@ -14,13 +12,11 @@ import (
 
 func CheckMigration() error {
 	stump.Log("checking if repo migration is needed...")
-	p := util.IpfsDir()
-	oldverB, err := ioutil.ReadFile(filepath.Join(p, "version"))
+	oldver, err := GetCurrentVersion()
```

Comment:
Erroring out when the "version" file is not there was wrong anyway, because the "version" file is created by some versions of IPFS only when `ipfs init` is run.